docs/content/preview/migrate/known-issues/mysql-oracle.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: MySQL and Oracle
+title: MySQL and Oracle source databases
 linkTitle: MySQL and Oracle
 headcontent: Known issues when migrating data from MySQL or Oracle.
 description: Refer to the known issues when migrating data using YugabyteDB Voyager and suggested workarounds.
```

Comment:
looks like the description that should be on known issues page; replace this one about mysql oracle